src/demo.py

```diff
@@ -0,0 +1,171 @@
+import time
```

Comment:
This file shouldn't be committed. If you want to keep it, move the file to somewhere else, commit the deletion, add `src/demo.py` to .gitignore, commit that, move the file back